hikari/impl/shard.py

```diff
@@ -120,99 +345,13 @@ class GatewayShardImplV6(shard.GatewayShard):
         `initial_is_afk`, and `initial_status` are not defined and left to their
         default values, then the presence will not be _updated_ on startup
         at all.
-
         If any of these _are_ specified, then any that are not specified will
         be set to sane defaults, which may change the previous status. This will
         only occur during startup, and is an artifact of how Discord manages
         these updates internally. All other calls to update the status of
         the shard will support partial updates.
     """
 
-    __slots__: typing.Sequence[str] = (
```

Comment:
Do we not need slots?

Reply:
Had this discussion with Dav, I don't think it provides much benefit, but this has been re-introduced for consistency.